### PR TITLE
D1451

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java [hunk 1]
```diff
@@ -261,7 +261,7 @@ private void populateAndVerifyLengthWindowCase(final KieBase kieBase) {
         this.kieSession.insert(ping3Event);
 
         this.kieSession.fireAllRules();
-        Assertions.assertThat(this.events).as("The rule should have fired for 2 events").containsExactly(ping2Event, ping3Event);
+        Assertions.assertThat(this.events).as("The rule should have fired for 2 events").contains(ping2Event, ping3Event);
```
Comment (tarilabs) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java]: I think this should verify only the last 2 events in the window, hence I believe it was more correct previous version?

File: drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java [hunk 2]
```diff
@@ -271,7 +271,7 @@ private void populateAndVerifyLengthWindowCase(final KieBase kieBase) {
         this.kieSession.insert(ping4Event);
 
         this.kieSession.fireAllRules();
-        Assertions.assertThat(this.events).as("The rule should have fired for ping event only").containsExactly(ping4Event);
+        Assertions.assertThat(this.events).as("The rule should have fired for ping event only").contains(ping4Event);
```
Comment (tarilabs) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java]: I think this should verify only the ping4Event is inside, and not the pongEvent,  hence I believe it was more correct previous version?

File: drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java [hunk 3]
```diff
@@ -388,7 +388,7 @@ private void populateAndVerifyTimeWindowCase(final KieBase kieBase) {
         clock.advanceTime(1, TimeUnit.SECONDS);
 
         this.kieSession.fireAllRules();
-        Assertions.assertThat(this.events).as("The rule should have fired for 2 events").containsExactly(ping2Event, ping3Event);
+        Assertions.assertThat(this.events).as("The rule should have fired for 2 events").contains(ping2Event, ping3Event);
```
Comment (tarilabs) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java]: Analogous, this is meant to verify for ping2Event and ping3Event, but not for the `pingEvent`,  hence I believe it was more correct previous version?

File: drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java [hunk 4]
```diff
@@ -400,7 +400,7 @@ private void populateAndVerifyTimeWindowCase(final KieBase kieBase) {
         clock.advanceTime(1, TimeUnit.SECONDS);
 
         this.kieSession.fireAllRules();
-        Assertions.assertThat(this.events).as("The rule should have fired for ping event only").containsExactly(ping4Event);
+        Assertions.assertThat(this.events).as("The rule should have fired for ping event only").contains(ping4Event);
```
Comment (tarilabs) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/oopath/OOPathCepTest.java]: Analogous, should indeed verify `ping4Event` is inside, but not anything else especially `pongEvent`,  hence I believe it was more correct previous version?

**Review by tarilabs — CHANGES_REQUESTED** [review]
If I recall correctly the semantic of `containsExactly`, I believe it was more correct the previous version.

**Comment by mariofusco** [comment]
ok to test

**Review by tarilabs — APPROVED** [review]
+1 disabled OOPath for `declared window` as described in JIRA DROOLS-1451 comment.

**Comment by mariofusco** [comment]
Replaced by https://github.com/droolsjbpm/drools/pull/1119

**Pull request closed without merging**